doc/classes/OS.xml

```diff
@@ -118,13 +136,28 @@
 				You can also incorporate environment variables using the [method get_environment] method.
 				You can set [code]editor/main_run_args[/code] in the Project Settings to define command-line arguments to be passed by the editor when running the project.
 				Here's a minimal example on how to parse command-line arguments into a dictionary using the [code]--key=value[/code] form for arguments:
-				[codeblock]
+				[codeblocks]
+				[gdscript]
 				var arguments = {}
 				for argument in OS.get_cmdline_args():
 				    if argument.find("=") &gt; -1:
 				        var key_value = argument.split("=")
 				        arguments[key_value[0].lstrip("--")] = key_value[1]
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				var arguments = new Godot.Collections.Dictionary();
+				foreach (var argument in OS.GetCmdlineArgs())
+				{
+				    if (argument.Find("=") &gt; -1)
+				    {
+				        string[] keyValue = argument.Split("=");
+				        // We don't have an LStrip StringExtension :( so we use Substring
```

Comment:
Can be fixed here after #43250 is merged. Same with HexEncode.

Reply:
forward-fixed